react/tic-tac-toe/src/app/screens/Game/components/Square/index.js

```diff
@@ -1,11 +1,20 @@
-import React from 'react';
+import React, {Component} from 'react';
+import {PropTypes} from 'prop-types';
```

Comment:
You can write import PropTypes from 'prop-types'  or import { func, string } from 'prop-types'